Improves authentication and profile updates

drodriguezatcci opened the pull request:
Improves user authentication flow and enhances profile update functionalities.

- Removes console logs for cleaner code.
- Sets `rememberMe` to true for email credentials during sign-in.
- Implements local storage to persist and remove email preferences.
- Integrates `authClient.getSession` to streamline the authentication verification process.

mdaniel543 committed:
Improves user registration and profile update

Enhances the registration process by adding phone number functionality and navigates users to the profile page upon successful registration.
Also, updates the profile page to cache user images for faster loading.
Additionally, the checkbox component includes label click handling.
Lastly, updates WhatsApp phone number env variable and includes a call option in the sell questions modal.

mdaniel543 committed:
Enhances user authentication and session management

Improves authentication flow by using better-auth session
management and removing custom auth service.

- Removes debugging console logs from auth routes.
- Updates login to persist 'remember me' email in localStorage.
- Streamlines session handling using authClient.
- Standardizes rememberMe value to true.

lraldaatcci commented on apps/portal-web/src/components/ui/CheckBox.tsx:
```diff
@@ -19,24 +19,33 @@ export const CheckBox: React.FC<CheckBoxProps> = ({
   isLabelLink = false,
   labelHref,
 }) => {
+  console.log("CheckBox - checked:", checked, "onChange:", typeof onChange);
```
Te sigue sirviendo ?

mdaniel543 committed:
Removes unnecessary console log

Removes a console log statement from the CheckBox component.

The console log was likely used for debugging and is no longer needed in the production code.
